StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/SheetViewController.swift

```diff
@@ -132,13 +132,15 @@ class SheetViewController: UIViewController {
         super.viewDidLayoutSubviews()
 
         if panePresentationStyle == .sheet {
-            let isLandscape = UIDevice.current.orientation.isLandscape
+            // Note that using `UIDevice.current.orientation.isLandscape`
+            // performed worse (/ was buggy) when testing on device
+            let isLandscapePhone = UIDevice.current.userInterfaceIdiom == .phone && (UIScreen.main.bounds.size.width > UIScreen.main.bounds.size.height)
```

Comment:
not sure what logic iOS is using to calculate `isLandscape` but it was performing noticeably worse 

Logically, however, I could see how this new code is better anyway: the layout logic, when boiled down to first-principles, isn't really about "landscape" but rather the fact that height got significantly decreased. The idea of "landscape" is just an abstraction away from what we are trying to do.